MdePkg/Library/StackCheckFailureHookLibNull/StackCheckFailureHook.c

```diff
@@ -0,0 +1,22 @@
+/** @file
+  Library provides a hook called when a stack cookie check fails.
+
+  Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
Please double check for this in all files.

```suggestion
  Copyright (c) Microsoft Corporation.
```